### DISCUSSION TITLE
Problem completing the Next.js Tutorial

### DISCUSSION THREAD

**Discussion by devtopher** [vercel/next.js#33590] [Help]
I'm new to Next.js and React and am running through the tutorial found here: https://nextjs.org/learn/basics/navigate-between-pages/pages-in-nextjs

The goal is to create a new page and then navigate to it here: http://localhost:3000/posts/first-post
The problem I had was that when I used the code supplied, I received this error: ```Error: The default export is not a React Component in page: "/posts/first-post"```
The code suggested was: 
```
export default function FirstPost() {
  return <h1>First Post</h1>
}
```
In order to make it work I had to add: 
```
import React from "react" 
```

Is this intended, because I did not see it in the tutorial? Is there a problem with my next.js install such that new files I create should have come with that portion already or is there something else I'm missing?

**Comment by uhAlexGrant** [comment]
Worked for me too. Thank you!

**Comment by JeomJeom** [comment]
I got the same error as well. 

I added this import statement `import Next from 'next'` and it works

**Comment by camden-kid** [comment]
I see the same error but that error does not appear for index.js which doesn't import React or Next.

**Comment by dallashomewatch** [comment]
I HATE it when tutorials have bad instructions!!!!!!!!!!!!!!
The new tutorial is giving errors too so I'm not able to complete it. Very frustrating.
There's a problem with .map for the routing of the 'posts'.

**Comment by ErikaSWang** [comment]
Two years, it still hasn't been fixed ... woof! :/
